File: Manage/Department.py
from Manage.Phone import Phone

class Department:
    def __init__(self, identfier: int, name: str, tele: Phone, location: str, chairPerson: str):
        self.identifier = int
        self.tele = tele
        self.location = location
        self.chair = chairPerson


    def unpackable(self):
        """
        get all attributes from a class programatically
        """

        return list(self.__dict__.values())

File: includes/classes/Department.php
<?php

//include_once("Geolocation.php");
include_once("Telephone.php");
include_once("Professor.php");


class Department {
    public $name;
    public $identifier; // Primary Key
    public $location;
    public $telephone;
    public $chairperson;

    function __construct($container_){
        // Assign the values of an array into the attributes of the class Department in order as seen above
        list($this->name, $this->identifier, $this->location, $this->telephone) = $container_;
        $this->telephone = new Telephone($this->telephone);
    }
}

?>
